src/Models/GFlasso.cpp

```diff
@@ -97,15 +97,15 @@ MatrixXd Gflasso::get_X(){
 }
 
 MatrixXd Gflasso::get_Y(){
-    return Y;
+    return y;
 };
 
 MatrixXd Gflasso::get_beta() {
     return beta;
 }
 
 void Gflasso::assertReadyToRun() {
-    throw runtime_error("Gflasso not implemented");
+    //throw runtime_error("Gflasso not implemented");
```

Comment:
I mean there are a couple functions in Gflasso class called "train". Do we want that, since I don't think PGD calls it? I thought that the workflow was model->setXY(), model->setAttributeMatrix(), alg->setUpRun(), alg->run(), alg->finishRun()